app/src/pages/inside/common/modals/editItemModal/editItemModal.jsx

```diff
@@ -235,14 +235,28 @@ export class EditItemModal extends Component {
     return URLS.testItemAttributeValuesSearch(projectId, item.launchId || item.id, key);
   };
 
+  isItemOwner = () => {
+    const {
+      data: { item, parentLaunch },
+      userId,
+    } = this.props;
+
+    if (item.owner) {
+      return userId === item.owner;
+    } else if (parentLaunch && parentLaunch.owner) {
+      return userId === parentLaunch.owner;
+    }
+
+    return true;
```

Comment:
Why exactly this meaning (`true`)? Do conditions act as negation?

Reply:
It means that launch (item) haven't any owner and can be edited by anyone.